paasta_tools/paasta_metastatus.py

```diff
@@ -495,7 +495,7 @@ def main():
     # Check to see if Chronos should be running here by checking for config
     try:
         chronos_config = load_chronos_config()
-    except ChronosNotConfigured:
+    except PaastaNotConfiguredError:
```

Comment:
The exception raised by `load_chronos_config` changed.  `load_chronos_config` calls `get_chronos_config` in `SystemPaastaConfig`, which raises a `PaastaNotConfiguredError`.

`load_chronos_config` could catch the `PaastaNotConfiguredError` and re-raise a `ChronosNotConfiguredError`, which might make more sense.